packages/react-components/react-datepicker/src/components/CalendarDayGrid/CalendarGridDayCell.tsx

```diff
@@ -85,10 +85,10 @@ export const CalendarGridDayCell: React.FunctionComponent<CalendarGridDayCellPro
       direction = -1;
     } else if (ev.key === ArrowDown) {
       targetDate = addWeeks(date, 1);
-    } else if (getRTLSafeKey(ArrowLeft, dir)) {
+    } else if (ev.key === getRTLSafeKey(ArrowLeft, dir)) {
```

Comment:
This is the cause of the month change when tabbing out of the day grid 🙃

Reply:
Ahhh, I'm very sorry for that, probably my fault...